FogCreek/Glitch-Community
# Second okra: fix invite bug

## Links
* https://second-okra.glitch.me
* https://glitch.manuscript.com/f/cases/3328764/

## Changes:
* previously we used to invite the current user instead of the user we were trying add, oh no! 

## How To Test:
* create a team, invite a user, notice what email that invite gets sent to, make doubly sure it's the one you think it should be!
